nixos/modules/services/misc/passopolis.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.passopolis;
+in
+{
+
+  ###### interface
+
+  options = {
+
+    services.passopolis = {
+
+      enable = mkOption {
+        default = false;
+        description = "
+          Enable passopolis password manager service.
+        ";
+      };
+
+      db_url = mkOption {
+        default = "jdbc:postgresql://localhost:5432/passopolis";
```

Comment:
If this url contains database passwords, it would be better if the url can be also stored outside of nix store in a file.